pyhmy/validator.py

```diff
@@ -0,0 +1,651 @@
+        rate: str
+            Max commission change rate of validator
+
+        Raises
+        ------
+        InvalidValidatorError
+            If input is invalid
+        """
+        try:
+            rate = Decimal(rate)
+        except InvalidOperation as e:
+            raise InvalidValidatorError(3, f'Max change rate must be a number') from e
+        if rate < 0:
+            raise InvalidValidatorError(3, f'Max change rate must be greater than or equal to 0')
+        if self._max_rate:
+            if rate > self._max_rate:
+                raise InvalidValidatorError(3, f'Max change rate must be less than or equal to max rate: {self._max_rate}')
+        else:
+            raise InvalidValidatorError(4, f'Max rate must be set before max change rate')
+        self._max_change_rate = rate.normalize()
+
+    def get_max_change_rate(self) -> Decimal:
+        """
+        Get validator max commission change rate
+
+        Returns
+        -------
+        Decimal
+            Validator max change rate
+        """
+        return self._max_change_rate
+
+    def set_rate(self, rate):
+        """
+        Set validator commission rate
+
+        Parameters
+        ----------
+        rate: str
+            Commission rate of validator
+
+        Raises
+        ------
+        InvalidValidatorError
+            If input is invalid
+        """
+        try:
+            rate = Decimal(rate)
+        except InvalidOperation as e:
+            raise InvalidValidatorError(3, f'Rate must be a number') from e
+        if rate < 0:
+            raise InvalidValidatorError(3, f'Rate must be greater than or equal to 0')
+        if self._max_rate:
+            if rate > self._max_rate:
+                raise InvalidValidatorError(3, f'Rate must be less than or equal to max rate: {self._max_rate}')
+        else:
+            raise InvalidValidatorError(4, f'Max rate must be set before rate')
+        self._rate = rate.normalize()
+
+    def get_rate(self) -> Decimal:
+        """
+        Get validator commission rate
+
+        Returns
+        -------
+        Decimal
+            Validator rate
+        """
+        return self._rate
+
+    def does_validator_exist(self, endpoint=_default_endpoint, timeout=_default_timeout) -> bool:
+        """
+        Check if validator exists on blockchain
+
+        Parameters
+        ----------
+        endpoint: :obj:`str`, optional
+            Endpoint to send request to
+        timeout: :obj:`int`, optional
+            Timeout in seconds
+
+        Returns
+        -------
+        bool
+            Does validator exist on chain
+
+        Raises
+        ------
+        RPCError, RequestsError, RequestsTimeoutError
+            If unable to get list of validators on chain
+        """
+        all_validators = get_all_validator_addresses(endpoint, timeout)
+        if self._address in all_validators:
+            return True
+        return False
+
+    def import_validator(self, info):
```

Comment:
`import` instead of `import_validator` as we use `export` and its already known that the object is a validator